ci-operator/jobs/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main-periodics.yaml

```diff
@@ -0,0 +1,83 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 6 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: stolostron
+    repo: acmqe-autotest
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: acm-ocp4.14-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.14-lp-interop-acm-interop-aws
```

Comment:
Slack config seems to be missing

Reply:
Yes it is good catch, i'll get this added.

Reply:
done